### PR TITLE
circleci 0.1.5607

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: Formula/circleci.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Circleci < Formula
   homepage "https://circleci.com/docs/2.0/local-cli/"
   # Updates should be pushed no more frequently than once per week.
   url "https://github.com/CircleCI-Public/circleci-cli.git",
-      :tag      => "v0.1.4786",
```
Comment (chenrui333) [inline Formula/circleci.rb]: last update was `Dec 26, 2018`

**Comment by chenrui333** [comment]
`brew test circleci` failed, I will fix it.

**Comment by chenrui333** [comment]
```
21:02:21 Error: circleci: failed
21:02:21 An exception occurred within a child process:
21:02:21   Test::Unit::AssertionFailedError: <0> expected but was
21:02:21 <2>.
21:02:21 /Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:55:in `block in assert_block'
21:02:21 /Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:1632:in `_wrap_assertion'
21:02:21 /Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:53:in `assert_block'
21:02:21 /Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:240:in `assert_equal'
```

**Comment by chenrui333** [comment]
relates to https://github.com/CircleCI-Public/circleci-cli/issues/256

**Comment by chenrui333** [comment]
Filing an issue for support the upgrade.

**Comment by zzak** [comment]
Just a heads up here, I've been meaning to come back to this. Sorry for the delay!

We should only need to add `make pack` command prior to running `go build` which will use the vendored `packr2` binary.

No more installing deps at compile time! :bow:

**Comment by chenrui333** [comment]
@zzak resuming the work on this PR now. :)

**Comment by chenrui333** [comment]
installed and test locally
```
$ brew test circleci
Testing circleci
==> /usr/local/Cellar/circleci/0.1.5607/bin/circleci version
==> /usr/local/Cellar/circleci/0.1.5607/bin/circleci build -c /private/tmp/circleci-test-20190516-44708-1pxzg4b/.circleci.yml 2>&1
==> /usr/local/Cellar/circleci/0.1.5607/bin/circleci help
==> /usr/local/Cellar/circleci/0.1.5607/bin/circleci update
```

**Pull request closed without merging**